Source/WebKit/UIProcess/Launcher/glib/ProcessLauncherGLib.cpp

```diff
@@ -125,7 +125,9 @@ static bool isSandboxEnabled(const ProcessLauncher::LaunchOptions& launchOptions
 
 void ProcessLauncher::launchProcess()
 {
+#if USE(LIBWPE) && ENABLE(BUBBLEWRAP_SANDBOX)
```

Comment:
```suggestion
#if ENABLE(BUBBLEWRAP_SANDBOX)
```

(This isn't related to libwpe.)

Reply:
:ok_hand:  thanks for the feedback